packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-optimize-ciemss-node.vue

```diff
@@ -151,10 +152,14 @@ watch(
 		const response = await pollResult(optId);
 		if (response.state === PollerState.Done) {
 			// Start 2nd simulation to get sample simulation from dill
-			const simulationIntervetions = await getOptimizedInterventions(optId);
+			const newInterventionResponse = await createInterventionPolicyFromOptimize(
+				modelConfigId.value ?? '',
+				optId
+			);
```

Comment:
At this stage, something catastrophic would've happened if you don't actually have `modelConfigId.value` right? 

Also it doesn't seem to make sense to create interventionPolicy with the blank model-config id??

If this is the case it would be better to phrase this as the follow if possible:
```
modelConfigId.value as string
```

It better indicates the intent here, as well as the situational awareness of the variables/states.

Reply:
Fair point will do